validator/accounts/v2/wallet_edit.go

```diff
@@ -21,7 +23,29 @@ func EditWalletConfiguration(cliCtx *cli.Context) error {
 	}
 	switch wallet.KeymanagerKind() {
 	case v2keymanager.Direct:
-		return errors.New("no configuration options available to edit for direct keymanager")
+		enc, err := wallet.ReadKeymanagerConfigFromDisk(ctx)
+		if err != nil {
+			return errors.Wrap(err, "could not read config")
+		}
+		cfg, err := direct.UnmarshalConfigFile(enc)
+		if err != nil {
+			return errors.Wrap(err, "could not unmarshal config")
+		}
+		log.Infof("Current configuration")
+		fmt.Printf("%s\n", cfg)
```

Comment:
Hmmm! Seems out of place

Reply:
This prints out the current configuration to stdout. Left a comment to clarify